avro-serializer/src/main/java/io/confluent/kafka/serializers/KafkaAvroSerializer.java

```diff
@@ -53,7 +63,21 @@ public void configure(Map<String, ?> configs, boolean isKey) {
       schemaRegistry = new CachedSchemaRegistryClient(
           (String) url, (Integer) maxSchemaObject);
     }
-
+    Object datumWriterClassName = configs.get(GENERIC_AVRO_DATUM_WRITER_CLASS);
```

Comment:
When #162 is merged, we'll be using confluentinc/common to parse configs, and that has support for specifying classes as settings and for creating instances automatically (see AbstractConfig#getConfiguredInstance). That might be a simpler solution than doing this all manually.